### PR TITLE
Add note about shell completion scripts

### PR DESCRIPTION
**Pull request by clubby789**
Add a note about the new shell completion scripts that will land in https://github.com/rust-lang/rust/pull/111388. cc @jyn514

### CODE REVIEW

File: src/building/suggested.md [hunk 1]
```diff
@@ -339,3 +339,8 @@ pkgs.mkShell {
   RUST_BOOTSTRAP_CONFIG = config;
 }
 ```
+
+## Shell Completions
+
+If you use Bash, Fish or PowerShell, you can find automatically-generated shell completion scripts for `x.py` in [`src/etc/completions`](https://github.com/rust-lang/rust/tree/master/src/etc/completions).
```
Comment (jyn514) [inline src/building/suggested.md:345]: Can we also document how to use the scripts? is it as simple as `source src/etc/completions/bash.sh`?

Comment (clubby789) [reply src/building/suggested.md:345]: I'll add some quick suggestions

**Comment by clubby789** [comment]
I think CI broke as part of Github outage here

**Comment by ChrisDenton** [comment]
```
Checking line lengths in all source files <= 100 chars...
Offending files and lines:
	src/building/suggested.md:348 : You can use `source ./src/etc/completions/x.py.<extension>` to load completions for your shell of choice,
1 offending lines found.
```

**Comment by clubby789** [comment]
Ah thank you, should be good now

**Review by jyn514 — APPROVED** [review]
amazing, thank you!

**Pull request merged**